src/mod.ts

```diff
@@ -11,6 +12,8 @@ export const projectScripts = () => {
         `deno run -q --unstable --config=deno.jsonc --allow-read --allow-write --allow-net https://deno.land/x/${BUILDER_TAG}/mod.ts`,
       "start":
         `deno run -q --unstable --config=deno.jsonc --allow-read --allow-net https://deno.land/x/${RUNTIME_TAG}/local-run.ts`,
+      "check-update":
+        `deno run -q --unstable --config=deno.jsonc --allow-read --allow-net https://deno.land/x/${HOOKS_TAG}/check_update.ts`,
```

Comment:
Should this be `check_update.ts` or `check-update.ts`? It looks like the filename uses a `-`:
```suggestion
        `deno run -q --unstable --config=deno.jsonc --allow-read --allow-net https://deno.land/x/${HOOKS_TAG}/check-update.ts`,
```
```suggestion
        `deno run -q --unstable --config=deno.jsonc --allow-read --allow-net https://deno.land/x/${HOOKS_TAG}/check_update.ts`,
```